docs/rules/prefer-stateless-function.md

```diff
@@ -43,3 +44,14 @@ class Foo extends React.Component {
   }
 }
 ```
+
+The following pattern is not considered warnings:
+
+```js
+class Foo extends React.PureComponent {
```

Comment:
Being a PureComponent does not make it a stateful component right? It just makes it benefit from `shouldComponentUpdate`